src/Mapper/ProgrammesDbToDomain/CoreEntityMapper.php

```diff
@@ -192,28 +353,28 @@ private function getClipModel(array $dbProgramme): Clip
         );
     }
 
-    private function getAncestryArray(array $dbProgramme, string $key = 'ancestry'): array
+    private function getAncestryArray(array $dbEntity, string $key = 'ancestry'): array
     {
         // ancestry contains a string of all IDs including the current one with
         // a trailing comma at the end (which makes it an empty item when exploding)
         // Thus we want an array of all but the final item (which is null)
-        $ancestors = explode(',', $dbProgramme[$key], -1) ?? [];
+        $ancestors = explode(',', $dbEntity[$key], -1) ?? [];
         return array_map(function ($a) {
             return (int) $a;
         }, $ancestors);
     }
 
-    private function getOptionsModel(array $dbProgramme, string $key = 'options'): Options
+    private function getOptionsModel(array $dbEntity, string $key = 'options'): Options
     {
         // ensure the full hierarchy has been fetched.
         // Options are only valid if we got everything.
-        if (!$this->hasFetchedFullHierarchy($dbProgramme)) {
+        if (!$this->hasFetchedFullHierarchy($dbEntity)) {
             return new UnfetchedOptions();
         }
 
-        // build for current dbProgramme the tree of options
-        $optionsTree = $this->getOptionsTree($dbProgramme, $key);
-        // get final options to be applied on programme from tree options hierarchy
+        // build for current dbGroup the tree of options
```

Comment:
Lots of stuff in this file reference 'dbGroup' and 'group', rather than 'dbEntity' / 'entity'